Solutions/GoogleWorkspaceReports/Data Connectors/GoogleWorkspaceTemplate_ccp/dcr.json

```diff
@@ -0,0 +1,115 @@
+[{
+  "name": "GoogleWorkspaceDCR",
+  "apiVersion": "2021-09-01-preview",
+  "type": "Microsoft.Insights/dataCollectionRules",
+  "location": "{{location}}",
+  "properties": {
+    "streamDeclarations": {
```

Comment:
streamDeclarations section is not required once standard table is deployed.
You can refer this, https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/AWS%20Security%20Hub/Data%20Connectors/AWSSecurityHubFindings_CCP/AWSSecurityHubFindings_DCR.json